step-capstone/src/components/TravelObjects/TravelObject.js

```diff
@@ -12,7 +12,10 @@ export default function TravelObject(props) {
     
     const handleClick = (event) => {
         setAnchorEl(event.currentTarget);
-        props.onClickObject(props.data.id);
+        // if has a placeId, zoom to marker on map
```

Comment:
great! so this will fix the problem we had last time: when user clicks in an object without an address, map breaks?

Reply:
Yes! :D